### PR TITLE
Fix date format tests

### PR DESCRIPTION
**Pull request by sssuneeth**

### CODE REVIEW

**Comment by sssuneeth** [comment]
@baseplate-admin

**Comment by baseplate-admin** [comment]
69

**Comment by baseplate-admin** [comment]
Dont

**Comment by sssuneeth** [comment]
Screenshot from 2023-04-25 22-49-49
@baseplate-admin

**Comment by baseplate-admin** [comment]
[Screencast from 04-26-2023 01:13:49 AM.webm](https://user-images.githubusercontent.com/61817579/234379539-2ea8dff5-460d-4dfa-86c2-a71827585626.webm)

File: src/lib/functions/format_time.ts [hunk 1]
```diff
@@ -11,12 +11,12 @@ export class formatTime {
...
 
 	public get formatSecondsToTimeStampDuration() {
-		const timeString = dayjs(this.#duration.asMilliseconds()).format('mm:ss');
+		const timeString = this.#duration.format('mm:ss');
 		return timeString;
 	}
 
 	public get formatSecondsToMinutes() {
-		const timeString = dayjs(this.#duration.asMilliseconds()).format('m');
+		const timeString = Math.floor(this.#duration.asMinutes());
```
Comment (baseplate-admin) [inline src/lib/functions/format_time.ts:19]: Dont use math function here :)

**Pull request closed without merging**